scripts/generate/templates/Checkpoints.t.js

```diff
@@ -86,10 +86,13 @@ function testPush(
 
     if (keys.length > 0) {
         ${opts.keyTypeName} lastKey = keys[keys.length - 1];
-        pastKey = _bound${capitalize(opts.keyTypeName)}(pastKey, 0, lastKey - 1);
-
-        vm.expectRevert();
-        this.push(pastKey, values[keys.length % values.length]);
+        if (lastKey > 0) {
+            ${opts.keyTypeName} maxKey = lastKey == 0 ? 0 : lastKey - 1;
+            pastKey = _bound${capitalize(opts.keyTypeName)}(pastKey, 0, maxKey);
```

Comment:
The `maxKey` variable is redundant because of the conditional.

```suggestion
            pastKey = _bound${capitalize(opts.keyTypeName)}(pastKey, 0, lastKey - 1);
```